python/splitter.py

```diff
@@ -20,6 +20,21 @@ def is_number(string):
     except ValueError:
         return False
 
+def validate_input(prompt):
+    #This function will take in a question prompt for an input and check to make sure it is a valid number.
+    #If it is not a number, it will ask the user to enter a valid number and re-prompt.
+    validated = False
+    while not validated:
+        try:
```

Comment:
seems like a duplicate of the is_number function maybe you can use that here? Also do you need to use a while loop over this?

Reply:
is_number returns true or false, while the validate_input returns a number input by the user

Reply:
Ah so you want to give the user another chance using this method if they mess up that makes sense

Reply:
the is number was more, can you use is number inside of validate input